hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHAWithData.java

```diff
@@ -109,8 +109,10 @@ public void testMultipartUpload() throws Exception {
   }
 
   @Test
-  public void testOMHAMetrics() throws InterruptedException,
-      TimeoutException, IOException {
+  public void testOMHAMetrics() throws Exception {
+    shutdown();
+    init();
+    getCluster().waitForClusterToBeReady();
```

Comment:
Instead of such tricks, I think it's better to move this test to a separate class.

Reply:
You are right. I made the changes.